src/api/config/application.rb

```diff
@@ -18,6 +18,7 @@ class Application < Rails::Application
     # config.autoload_paths += %W(#{config.root}/extras)
     config.eager_load_paths << Rails.root.join('lib', 'backend')
 
+    # Make sure all routes are loaded and automatically reloaded (specially in development)
```

Comment:
I wouldn't change this file. It looks like you tested something and forgot to remove those changes.